README.md

```diff
@@ -158,7 +158,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [Codacy](https://www.codacy.com) [:rocket:](https://www.codacy.com/careers) | Automated code reviews & code analytics. | `Lisboa` `Remote` |
-| [MemSQL](https://www.memsql.com/) [:rocket:](https://www.memsql.com/careers/jobs/) | A Data Platform Purpose-Built for Breakthroughs. | `Lisboa` |
+| [Singlestore](https://www.singlestore.com/) (formerly MemSQL) [:rocket:](https://www.singlestore.com/careers/jobs/) | Distributed, relational database that handles both transactions and real-time analytics at scale. | `Lisboa` |
```

Comment:
I think the `(formerly MemSQL)` is unnecessary but if you think it's important I think it's better to place on the description instead.